pkg/ingester/ingester.go

```diff
@@ -113,6 +115,8 @@ const (
 	// Value used to track the limit between sequential and concurrent TSDB opernings.
 	// Below this value, TSDBs of different tenants are opened sequentially, otherwise concurrently.
 	maxTSDBOpenWithoutConcurrency = 10
+
+	shutdownMarkerFile = "shutdown-requested.txt"
```

Comment:
super-nit: `shutdownMarkerFilename` (but I wouldn't be surprised if we're not consistent about it)